tools/c7n_mailer/c7n_mailer/cli.py

```diff
@@ -58,6 +58,7 @@
         'redis_port': {'type': 'integer'},
         'datadog_api_key': {'type': 'string'},              #TODO: encrypt with KMS?
         'datadog_application_key': {'type': 'string'},      #TODO: encrypt with KMS?
+        'slack_token': {'type': 'string'},
```

Comment:
do we also need a slack domain here? or is the token specific to a domain?

Reply:
The token is unique to a particular environment so a different token would be needed for each workspace, similar to how the Datadog API key works.